test/test.variadic.args.js

```diff
@@ -26,7 +26,10 @@ program.args.should.have.lengthOf(3);
 program.args[0].should.eql('arg0');
 program.args[1].should.eql(['arg1', 'arg2', 'arg3']);
 
-program
+//Create fresh instance
+var program2 = new program.Command();
+
+program2
```

Comment:
same sub command was registered twice.
Previously it work because events was overridden.
But inside `commands` array it was two instances with same name.
This lead to problem with help messages and should be handled in separate commit.
IMHO, exception should be thrown then duplicated commands registered.